setup.md

```diff
@@ -5,6 +5,11 @@ nav_title: Flutter Setup
 permalink: /setup/
 ---
 
+(Please remember, Flutter is in _technology preview_. Thanks
+for trying it out, and please don't hesitate to
+[file issues or bugs](https://github.com/flutter/flutter/issues/new)
+for us.)
```

Comment:
Should we have this note on every page?  Maybe we should incorporate it into the banner at the top of the page somehow.